docs/DeveloperGuide.md

```diff
@@ -282,18 +291,22 @@ However, this requires there to be no duplicate dog or program names.
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | Dog school manager   | Create a list of dogs and their respective owners in the dog school        | Keep track of the dogs we are responsible for     |
-| `* * *`  | Busy owner          | Delete dog profiles and owner profiles            | Remove dogs or owners that are no longer in the school                                                           |
-| `* * *`  | Dog school manager   | Add dog profiles and owner profiles             |  Add new dogs or owners that join the school         |
-| `* * *`  | Dog school manager   | Get instructions         | Get help when I do not know how to use the application |
-| `* * *`  | Dog school manager   | Exit the application   |           |
-| `* *`    | Dog school manager   | Edit a dog profile when the information is wrong/outdated |                                             |
-| `* *`    | Dog school manager   | Create a dog program for the dog students            |                                                  |
-| `* *`    | Dog school manager   | Enrol dogs into a specific dog program | Dogs who recently joined a program are added to the class list |
-| `* *`    | Dog school manager   | Drop dogs out of a specific dog program | Dogs that have left the class are no longer in the class list  |
-| `* *`    | Advanced user        | Edit in bulk quickly without having to be familiar with the app | Minimize chance of someone else seeing them by accident |
+| Priority | As a …​           | I want to …​                                                            | So that I can…​                    |
+| -------- | -------------------- | -------------------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| `* * *`  | Dog school manager   | Have a list of dogs, owners, and programs in the school                    | Keep track of the operations and dogs we are responsible for     |
+| `* * *`  | Dog school manager   | Easily switch between the different lists for dogs, owners and programs    | Quickly view all the profiles of one entity type     |
+| `* * *`  | Dog school manager   | Add dog/owner/program profiles                                             | Keep track of the operations and parties involved in the school|
+| `* * *`  | Dog school manager   | Delete dog/owner/program profiles                                          | Keep track of the operations and parties involved in the school|
+| `* * *`  | Dog school manager   | Edit a dog/owner/program profile                                           | Correct/update a profile when it is incorrect.|
```

Comment:
you ended the sentence with a fullstop here but not the other user stories. should choose either all fullstops or none for consistency